nilearn/plotting/surface/_utils.py

```diff
@@ -87,16 +91,16 @@ def _check_bg_map(bg_map, hemi):
 
 
 def _get_hemi(mesh, hemi):
-    """Check that a given hemisphere exists in a PolyMesh and return the
-    corresponding mesh. If "both" is requested, combine the left and right
-    hemispheres.
+    """Check that a given hemisphere exists in a
+    :obj:`~nilearn.surface.PolyMesh` and return the corresponding mesh. If
+    "both" is requested, combine the left and right hemispheres.
     """
     if hemi == "both":
         return combine_hemispheres_meshes(mesh)
     elif hemi in mesh.parts:
         return mesh.parts[hemi]
     else:
-        raise ValueError("hemi must be one of left, right or both.")
+        raise ValueError("hemi must be one of 'left', 'right' or 'both'.")
 
 
 def check_surface_plotting_inputs(
```

Comment:
Could we remove the parameters `map_var_name` and `mesh_var_name`? Is it really important to display to user if it is `roi_map` or `surf_map`? We can simply say `map` and `mesh`?

Reply:
sure: but I was just trying to make it easier for users to figure out which parameters they have passed in public functions is "wrong"

Reply:
`mesh_var_name` only takes value `surf_mesh` in the code. Map can be either `roi`, `surf` or `stat`. If you think it is not clear enough for the user just to say `map`, I'll keep it.